src/main/java/seedu/address/commons/util/StringUtil.java

```diff
@@ -29,19 +32,46 @@ public static boolean containsWordIgnoreCase(String sentence, String word) {
         String preppedWord = word.trim();
         checkArgument(!preppedWord.isEmpty(), "Word parameter cannot be empty");
         checkArgument(preppedWord.split("\\s+").length == 1, "Word parameter should be a single word");
-
-        String preppedSentence = sentence;
+        String preppedSentence = sentence.trim();
         String[] wordsInPreppedSentence = preppedSentence.split("\\s+");
 
-        for (String wordInSentence: wordsInPreppedSentence) {
-            if (wordInSentence.equalsIgnoreCase(preppedWord)) {
+        for (String wordInSentence : wordsInPreppedSentence) {
+            if (wordInSentence.toLowerCase().contains(preppedWord.toLowerCase())) {
```

Comment:
Why not convert `preppedWord` and `preppedSentence` into lowercase outside of the loop? It still makes sense to call them "prepped" when you're trimming and converting them into lowercase, since you're preparing them for this comparison. It makes the loop easier to read, since you don't need to keep converting them into lowercase in each iteration.